lib/graphql_rails/errors/validation_error.rb

```diff
@@ -18,5 +18,16 @@ def type
     def to_h
       super.merge('field' => field, 'short_message' => short_message)
     end
+
+    private
+
+    def humanized_field(field)
+      return if field.blank?
+
+      stringified_field = field.to_s
+      return if stringified_field == 'base'
```

Comment:
:sparkles: Would it make sense to declare `base` as constant?

```ruby
      BASE_FIELD_NAME = 'base'

      return if stringified_field == BASE_FIELD_NAME
```

Reply:
[fixed](https://github.com/samesystem/graphql_rails/pull/238/commits/593597a4c2251312d7eb24408777b7676b72fb3a)